answer/shinooooo/JS/08_createDiv/createDiv.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <script>
+    let i= 0;
+    createDiv = () => {
```

Comment:
```suggestion
    const createDiv = () => {
```
アロー関数を定義する時は変数宣言をつけましょう:eyes: